setup/resources/Microsoft.Portable.FSharp.Shim.targets

```diff
@@ -1,5 +1,6 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  
-  <Import Project="$(MSBuildProgramFiles32)\Microsoft SDKs\F#\10.1\Framework\v4.0\Microsoft.Portable.FSharp.Targets" />
-  
+
+  <Import Condition="'$(FSharpCompilerPath)' != ''" Project="$(FSharpCompilerPath)\Microsoft.Portable.FSharp.Targets" />
+  <Import Condition="'$(FSharpCompilerPath)' == ''" Project="$(MSBuildProgramFiles32)\Microsoft SDKs\F#\10.1\Framework\v4.0\Microsoft.Portable.FSharp.Targets" />
```

Comment:
But nothing will install to this path anymore, since we removed the MSI?

Reply:
That is fixed in this PR:  https://github.com/Microsoft/visualfsharp/pull/4819

Here:https://github.com/Microsoft/visualfsharp/pull/4819/files#diff-3c8a247250b0ee95f17ac1a0253f962fL1

Reply:
I'll have to rebase that PR to fix this bottom line, but this will still work.  The MSI removal doesn't remove these files, they're just placed under `Common7\IDE\...`.